.dockerignore

```diff
@@ -0,0 +1,29 @@
+*.pyc
+include/
+lib/
+staticfiles/
+media/
+.build
+
+# sqlite3 db
+*.db
+*.sqlite3
+
+# sublime stuff
+*.sublime-workspace
+*.sublime-project
```

Comment:
I'd argue things about people's specific working environments shouldn't be in our project-level gitignore files but, rather, in their own user-level gitignores? (This goes for `.DS_Store` later, too.)

That way, we're using gitignore to document the stuff that *our project* creates that we don't want to be committable.